docs/docs/reference/project-files/connectors.md

```diff
@@ -163,12 +162,11 @@ _[boolean]_ - Allow access to host environment configuration
 # Example: Azure connector configuration
 type: connector # Must be `connector` (required)
 driver: azure # Must be `azure` _(required)_
-azure_storage_account: "mystorageaccount" # Azure storage account name  
-azure_storage_key: "credentialjsonstring" # Azure storage access key  
-azure_storage_sas_token: "optionaltoken" # Optional SAS token for authentication  
-azure_storage_connection_string: "optionalconnectionstring" # Optional connection string  
-azure_storage_bucket: "mycontainer" # Azure Blob Storage container name _(required)_  
-allow_host_access: true # Allow host environment access
+azure_storage_account: "mystorageaccount" # Azure storage account name   _(required)_
+azure_storage_key: "credentialstring" # Azure storage access key   _(required)_
+# azure_storage_sas_token: "optionaltoken"         # Optional SAS token for authentication  
+# azure_storage_connection_string: "optionalconnectionstring" # Or use the connection string instead of storage account and key
+azure_storage_bucket: "mycontainer" # Azure Blob Storage container name  
```

Comment:
<img width="988" height="200" alt="image" src="https://github.com/user-attachments/assets/127ae785-0761-4cdd-82eb-a4bf01bdaf0d" />

Is this how it's intended to look? It's a bit hard to read with
1. `azure_storage_sas_token` and `azure_storage_connection_string` commented out.
2. The weird indentation of the "# Optional SAS token..."` comment